### PR TITLE
Bluespace belts

### PR DESCRIPTION
**Pull request by TheFakeElon**
## About The Pull Request

Ports bluespace belts from paradise

## Why It's Good For The Game

Provides more tool storage options.

## Changelog
:cl:
add: adds bluespace belts
/:cl:

### CODE REVIEW

File: austation/code/game/objects/items/storage/belt.dm [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+
+// -------------------------------------
+//     Bluespace Belt
+// -------------------------------------
+/obj/item/storage/belt/bluespace
+	name = "Belt of Holding"
+	desc = "The greatest in pants-supporting technology."
+	icon_state = "holdingbelt"
+	item_state = "holdingbelt"
```
Comment (MCterra10) [inline austation/code/game/objects/items/storage/belt.dm:11]: You've got two icon_state definitions.
Also, you need to add a `icon = 'path'` var pointing to the icon file you added

File: austation/code/modules/research/designs/bluespace_designs.dm [hunk 2]
```diff
@@ -0,0 +1,12 @@
+/////////////////////////////////////////
+//////////////Bluespace//////////////////
+/////////////////////////////////////////
+/datum/design/bluespace_belt
+	name = "Belt of Holding"
+	desc = "An astonishingly complex belt popularized by a rich blue-space technology magnate."
+	id = "bluespace_belt"
+	req_tech = list("bluespace" = 7, "materials" = 5, "engineering" = 5, "plasmatech" = 6)
```
Comment (MCterra10) [inline austation/code/modules/research/designs/bluespace_designs.dm:8]: This is an incompatible research system. Paradise doesn't use techwebs. Please add belts to a techweb node and change code accordingly

File: austation/code/modules/research/designs/bluespace_designs.dm [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+	build_type = PROTOLATHE
+	materials = list(MAT_GOLD = 1500, MAT_DIAMOND = 3000, MAT_URANIUM = 1000)
```
Comment (MCterra10) [inline austation/code/modules/research/designs/bluespace_designs.dm:10]: Datum mats, not defines please

File: beestation.dme [hunk 4]
```diff
@@ -18,6 +18,7 @@
 
 // BEGIN_INCLUDE
 #include "_maps\_basemap.dm"
+#include "austation\define_includes.dm"
```
Comment (MCterra10) [inline beestation.dme:21]: Remove this

File: austation/code/game/objects/items/storage/belt.dm [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+	storage_slots = 14
+	w_class = WEIGHT_CLASS_BULKY
+	max_w_class = WEIGHT_CLASS_SMALL
+	max_combined_w_class = 21 // = 14 * 1.5, not 14 * 2.  This is deliberate
+	origin_tech = "bluespace=5;materials=4;engineering=4;plasmatech=5"
```
Comment (MCterra10) [inline austation/code/game/objects/items/storage/belt.dm:16]: Techwebs. Fix it

File: austation/code/game/objects/items/storage/belt.dm [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+	can_hold = list()
```
Comment (MCterra10) [inline austation/code/game/objects/items/storage/belt.dm:17]: What does this mean? It can hold all small items?

**Comment by TheFakeElon** [comment]
fug

**Comment by TheFakeElon** [comment]
travis REEEEEEEEEEEEEEEEE my local works fine

**Comment by MCterra10** [comment]
I'll rerun the check, it could be a fluke

**Comment by TheFakeElon** [comment]
thank you travis

**Comment by MCterra10** [comment]
not seeing any issues

**Pull request merged**